### PR TITLE
Add missed public modifiers, fix warning

### PR DESCRIPTION
**Pull request by astrokin**
## Checklist

- [ ] New Extension
- [ ] New Test
- [ x] Changed more than one extension, but all changes are related to visibility modifiers
- [x] Trivial change (doesn't require changelog)

### CODE REVIEW

File: Sources/StringExtensions.swift [hunk 1]
```diff
@@ -395,14 +395,14 @@ extension String {
...
-    func height(_ width: CGFloat, font: UIFont, lineBreakMode: NSLineBreakMode?) -> CGFloat {
+    public func height(_ width: CGFloat, font: UIFont, lineBreakMode: NSLineBreakMode?) -> CGFloat {
         var attrib: [String: AnyObject] = [NSFontAttributeName: font]
         if lineBreakMode != nil {
             let paragraphStyle = NSMutableParagraphStyle()
             paragraphStyle.lineBreakMode = lineBreakMode!
             attrib.updateValue(paragraphStyle, forKey: NSParagraphStyleAttributeName)
         }
-        let size = CGSize(width: width, height: CGFloat(DBL_MAX))
+        let size = CGSize(width: width, height: CGFloat(Double.greatestFiniteMagnitude))
```
Comment (Khalian) [inline Sources/StringExtensions.swift]: The warning is addressed in https://github.com/goktugyil/EZSwiftExtensions/pull/403. Would you mind rebasing your changes to latest mainline branch ?

**Comment by Khalian** [comment]
Erm, rebase not merge. And could you please squash the commits into one as well please ?

**Comment by astrokin** [comment]
@Khalian what's the problem?

**Comment by Khalian** [comment]
A rebase is a replay of your work on top of mainline. A merge is creating an extra commit which has two parents - the existing mainline head and the head of your branch. 

In effect a merge commit defines "how" a change was arrived at. We already have that information in the PR. Hence it makes sense to only keep "what" changes went in, which is a rebase.

**Pull request merged**